kornia/optical_flow/lucas_kanade_opticalflow.py

```diff
@@ -0,0 +1,80 @@
+import torch
+import torch.nn as nn
+
+from kornia.filters import spatial_gradient
+
+
+def optical_flow_lk(image_prev: torch.Tensor, image_next: torch.Tensor, window_size: int) -> torch.Tensor:
```

Comment:
Can you provide any reference for this implementation so that we can compare bitwise ?